package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "asab_webui_components",
-	"version": "25.2.12",
+	"version": "25.2.13",
```

Comment:
I consider this as a significant change, so it should be something like `25.3.1`